host/xtest/pkcs11_1000.c

```diff
@@ -2565,3 +2565,218 @@ static void xtest_pkcs11_test_1011(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(pkcs11, 1011, xtest_pkcs11_test_1011,
 		 "PKCS11: Test Find Objects");
+
+static void xtest_pkcs11_test_1012(ADBG_Case_t *c)
+{
+	CK_RV rv = CKR_GENERAL_ERROR;
+	CK_SLOT_ID slot = 0;
+	CK_SESSION_HANDLE session = CK_INVALID_HANDLE;
+	CK_FLAGS session_flags = CKF_SERIAL_SESSION | CKF_RW_SESSION;
+	CK_OBJECT_HANDLE obj_hdl = CK_INVALID_HANDLE;
+	size_t i = 0;
+
+	CK_OBJECT_CLASS objClass = CKO_DATA;
+	CK_BBOOL objToken = CK_FALSE;
+	CK_BBOOL objPrivate = CK_FALSE;
+	uint8_t objValue[5] = { 1, 2, 3, 4, 5 };
+	const char *objLabel = "Label";
+
+	CK_ATTRIBUTE object_template[] = {
+		{ CKA_CLASS, &objClass, sizeof(CK_OBJECT_CLASS) },
+		{ CKA_TOKEN, &objToken, sizeof(CK_BBOOL) },
+		{ CKA_PRIVATE, &objPrivate, sizeof(CK_BBOOL) },
+		{ CKA_VALUE, objValue, sizeof(objValue) },
+		{ CKA_LABEL, (CK_UTF8CHAR_PTR)objLabel, strlen(objLabel) },
+	};
+
+	CK_BBOOL gToken = CK_TRUE;
+	CK_BBOOL gPrivate = CK_TRUE;
+	CK_OBJECT_CLASS gClass = ~0;
+	uint8_t gValue[128] = { 0 };
+
+	CK_ATTRIBUTE get_attr_template_bc[] = {
+		{ CKA_TOKEN, &gToken, sizeof(gToken) },
+		{ CKA_CLASS, &gClass, sizeof(CK_OBJECT_CLASS) },
+	};
+
+	CK_ATTRIBUTE get_attr_template_cb[] = {
+		{ CKA_CLASS, &gClass, sizeof(CK_OBJECT_CLASS) },
+		{ CKA_TOKEN, &gToken, sizeof(gToken) },
+	};
+
+	CK_ATTRIBUTE get_attr_template_ve[] = {
+		{ CKA_VALUE, &gValue, sizeof(objValue) },
+	};
+
+	CK_ATTRIBUTE get_attr_template_vl[] = {
+		{ CKA_VALUE, &gValue, sizeof(gValue) },
+	};
+
+	CK_ATTRIBUTE get_attr_template_bvecb[] = {
+		{ CKA_TOKEN, &gToken, sizeof(gToken) },
+		{ CKA_VALUE, &gValue, sizeof(objValue) },
+		{ CKA_CLASS, &gClass, sizeof(gClass) },
+		{ CKA_TOKEN, &gPrivate, sizeof(gPrivate) },
+	};
+
+	CK_ATTRIBUTE get_attr_template_bvlcb[] = {
+		{ CKA_TOKEN, &gToken, sizeof(gToken) },
+		{ CKA_VALUE, &gValue, sizeof(gValue) },
+		{ CKA_CLASS, &gClass, sizeof(gClass) },
+		{ CKA_TOKEN, &gPrivate, sizeof(gPrivate) },
+	};
+
+	rv = init_lib_and_find_token_slot(&slot);
+	if (!ADBG_EXPECT_CK_OK(c, rv))
+		return;
+
+	rv = init_test_token(slot);
+	if (!ADBG_EXPECT_CK_OK(c, rv))
+		return;
```

Comment:
From here, aborts shall goto:
```c
close_lib:
	ADBG_EXPECT_CK_OK(c, close_lib());
}

Reply:
Ok. Will fix.

Note: it was a copy from other test case in the same file.